typescript/scripts/README.md

```diff
@@ -0,0 +1,15 @@
+# scripts
```

Comment:
Update this README, it should explain what this package is and what scripts are available

Reply:
done